runtime/include/gpu/rocm/chpl-gpu-gen-includes.h

```diff
@@ -94,12 +94,9 @@ __host__ static inline void chpl_assert_on_gpu(int32_t lineno, int32_t filenameI
 }
 
 __device__ static inline unsigned int chpl_gpu_clock(void) {
-  // TODO
-  chpl_gpu_write("chpl_gpu_clock not implemented for ROCM gpu runtime\n");
-  return -1;
+  return (unsigned int)clock();
 }
 __host__ static inline unsigned int chpl_gpu_clock(void) {
-  chpl_internal_error("chpl_gpu_clock not implemented for ROCM gpu runtime");
```

Comment:
This should probably remain but have different wording, right? "chpl_gpu_clock` called from a CPU" or something.

Reply:
I think it's debatable what we should do.

Consider some kernel like this:

```chpl

on maybeGpuMaybeCpu() {
  startTime = gpuClock();
  // some computation 
  endTime = gpuClock();
}
```

Do we want `gpuClock()` to error out when not run on the GPU or do we want it to return 0 (indicating that no time was spent on the GPU).

Reply:
I think that code exactly should be an error. But maybe what you meant is something more like:

```chapel

on maybeGpuMaybeCpu() {
  foreach i in 1..n {
    startTime = gpuClock();
    // some computation 
    endTime = gpuClock();
  }
}
```

Down the road, I want our regular `stopwatch` to be capable of doing this, so hopefully this is a moot conversation. But even with that I am on the "let's error" side of the fence mostly because `gpu` is in the name and invoking this from the host indicates something going wrong, I would think.

That being said, what you have in this PR is consistent with what we have for CUDA it looks like, design will change as I hope to make this feature a more first-class via stopwatch some day, this is an important PR that I don't want us to get more distracted by this.

Reply:
Yeah that is the example I meant. And yes I agree having our regular `stopwatch` handle it is what we'll want in the long term, this and `gpuWriteln()` are really short-term solutions that exist until we can get something working long term.